lxqt-leave/main.cpp

```diff
@@ -106,6 +106,6 @@ int main(int argc, char *argv[])
                 Qt::AlignCenter,
                 dialog.size(),
                 qApp->desktop()->screenGeometry(QCursor::pos())));
-    dialog.setMaximumSize(dialog.minimumSize());
+    dialog.setMaximumSize(dialog.size());
```

Comment:
I would rather use the `sizeHint()`... and if we want more spacing, it should be done by the layout spacings in the .ui.

...I will push a commit instantly